### PR TITLE
Covid chatbot pmidc team

### PR DESCRIPTION
**Pull request by egovmis1**
fetch and merge

### CODE REVIEW

File: xstate-chatbot/nodejs/src/machine/vitals.js [hunk 1]
```diff
@@ -1010,18 +1021,31 @@ const vitalsFlow = {
...
           actions: assign((context, event) => {
             let mediaMessage = mediaUtil.createMediaMessage(`${config.staticMediaPath}/controlRoom_contact_numbers`, 'jpeg', undefined, '');
             dialog.sendMessage(context, dialog.get_message(messages.addVitals, context.user.locale), false);
             dialog.sendMessage(context, mediaMessage);
           }),
           target: '#endstate'
-        }
+        },
+        {
+          cond: (context, event) =>context.slots.person.rrt==='YES',
```
Comment (rushang7-eGov) [inline xstate-chatbot/nodejs/src/machine/vitals.js:1036]: When for both the conditions you want to do the same action, you don't need to write it separately.

File: xstate-chatbot/nodejs/src/machine/vitals.js [hunk 2]
```diff
@@ -1010,18 +1021,31 @@ const vitalsFlow = {
     addVitals: {
       id: 'addVitals',
       invoke: {
-        src: (context) => vitalsService.addVitals(context.user, context.slots.vitals),
-        onDone: {
+        src: (context) => vitalsService.addVitals(context.user, context.slots.vitals,context.slots.registerPatient),
+        onDone: [
+        {
+          cond: (context, event) =>context.slots.person.rrt==='NO',
```
Comment (rushang7-eGov) [inline xstate-chatbot/nodejs/src/machine/vitals.js:1027]: Just remove cond from here. That will execute the action without checking for any condition

**Review by rushang7-eGov — COMMENTED** [review]
@egovmis1 Please make the suggested changes

File: xstate-chatbot/nodejs/src/machine/vitals.js (hunk 1 above)
Comment (egovmis1) [reply xstate-chatbot/nodejs/src/machine/vitals.js:1036]: For both condition dialog.get_message is different

File: xstate-chatbot/nodejs/src/machine/vitals.js (hunk 2 above)
Comment (egovmis1) [reply xstate-chatbot/nodejs/src/machine/vitals.js:1027]: For both condition dialog.get_message is different

**Pull request merged**